var/spack/repos/builtin/packages/nvpl-fft/package.py

```diff
@@ -36,3 +40,25 @@ def url_for_version(self, version):
 
     def install(self, spec, prefix):
         install_tree(".", prefix)
+
+    @property
+    def headers(self):
+        return find_all_headers(self.spec.prefix.include)
+
+    @property
+    def libs(self):
+        return find_libraries("libnvpl_fftw", self.spec.prefix.lib, shared=True, recursive=True)
+
+    @run_after("install")
+    def fix_include(self):
+        # include/nvpl_fftw/ is on;y present in version 0.4 and later
+        if self.spec.satisfies("@:0.3"):
+            return
```

Comment:
The `@when` decorator does not appear to be compatible with the `@run_after` decorator.

Reply:
run_after accepts it as a kwarg `when=...`